doc/getting-started/launching-a-testnet.md

```diff
@@ -0,0 +1,103 @@
+### Prerequisites
+You must already be able to build everything in the `cardano-node` repository.
+
+## Running the chairman test
+The chairman test will run some testnets and perform some basic tests to ensure they
```

Comment:
Does this mean that the chairman test will:

0. Run several testnets
1. For each of these testnets, perform basic tests

?

Reply:
Yes, this is right.

Reply:
@newhoggy Can we link to what the chairman is testing/does?

Reply:
Do we have existing chairman test documentation that can be re-used?  The actually chairman test is identical to what it was before.  The main difference with the new tests is the way it sets up the testnet for testing.

Reply:
I added some more details.